mimic/model/flavor_objects.py

```diff
@@ -0,0 +1,76 @@
+"""
+Model objects for mimic flavors.
+"""
+
+from characteristic import attributes
+
+
+@attributes(['flavor_id', 'name', 'ram', 'tenant_id'])
+class Flavor(object):
+    """
+    A Flavor object
+    """
+
+    static_defaults = {
+        "vcpus": 20,
+        "swap": "",
+        "rxtx_factor": 10000.0,
+        "OS-FLV-EXT-DATA:ephemeral": 0,
+        "disk": 800
+    }
+
+    def links_json(self, absolutize_url):
+        """
+        Create a JSON-serializable data structure describing the links to this
+        flavor.
+        """
+        return [
+            {
+                "href": absolutize_url("v2/{0}/flavors/{1}"
+                                       .format(self.tenant_id, self.flavor_id)),
+                "rel": "self"
+            },
+            {
+                "href": absolutize_url("{0}/flavors/{1}"
+                                       .format(self.tenant_id, self.flavor_id)),
+                "rel": "bookmark"
+            }
+        ]
+
+    def extra_specs_json(self):
+        """
+        Create a JSON-serializable data structure describing
+        ``OS-FLV-WITH-EXT-SPECS:extra_specs`` for a flavor.
+        """
+        return {
+            "quota_resources": "instances=mimic-instances,ram=mimic-ram",
+            "class": "mimic",
```

Comment:
Flavor classes are purely a Rackspace construct, I'm not sure if that's desired in mimic or not, but wanted to point it out. Might be the same for all of these extra specs.

That said, extra specs are a completely arbitrary blob, so maybe it's fine since a Nova operator can put anything here. :)